src/actions/FioActions.js

```diff
@@ -51,22 +55,47 @@ export const refreshConnectedWallets = async (dispatch: Dispatch, getState: GetS
 }
 
 export const checkFioObtData = (walletId: string, transactions: EdgeTransaction[]) => async (dispatch: Dispatch, getState: GetState) => {
+  if (checkFioObtDataCounter > MAX_OBT_DATA_CHECKS) return // to prevent loop if account does not have fio wallets
```

Comment:
This seems pretty arbitrary and could mean that FIO obt data isn't checked after some time. Can you explain the rationale for 15 or is there a better way to determine if FIO names are present and this function can exit early?

Reply:
there is no important meaning of the value, just 15 * 400 is 6 seconds, and it is just an assumption that wallets should be loaded in this time. as I remember it all was removed before because we wanted to refactor this logic to be in the `core` lib, but for some reason it was forgotten. not sure we should rewrite the logic in current PR, is it ok to create separate task for it?